google/cloud/bigquery/schema.py

```diff
@@ -66,6 +66,46 @@ class _DefaultSentinel(enum.Enum):
 _DEFAULT_VALUE = _DefaultSentinel.DEFAULT_VALUE
 
 
+class FieldElementType(object):
+    """Represents the type of a field element.
+
+    Args:
+        element_type (str): The type of a field element.
+    """
+
+    def __init__(self, element_type: str):
+        self._properties = {}
+        self._properties["type"] = element_type.upper()
+
+    @property
+    def element_type(self):
```

Comment:
Do you also need a setter for this property?  Or you're only exposing a setter from the containing type?

Reply:
I decided to not allow the element type to be altered in order to protect the table schema from being changed unintentionally. As a reference, most of the properties in class `SchemaField` don't have setters either.